contents/serializers.py

```diff
@@ -13,4 +13,41 @@ class Meta:
 class CommentSerializer(serializers.ModelSerializer):
     class Meta:
         model = Comment
-        fields = '__all__'
\ No newline at end of file
+        fields = '__all__'
+
+# class ArticleSerializerlike(serializers.ModelSerializer):
+#     liked_by_user = serializers.SerializerMethodField()
+
+#     class Meta:
+#         model = Article
+#         fields = ['id', 'author', 'content', 'image', 'created_at','updated_at', 'liked_by_user']
+
+#     def get_liked_by_user(self, obj):
+#         request = self.context.get('request')
+#         if request and request.user.is_authenticated:
+#             return obj.like_user.filter(pk=request.user.pk).exists()
+#         return False
+    
+class Article_like_comment_Serializer(serializers.ModelSerializer):

```

Comment:
python에서 class name은 camel case를 따릅니다~~
```suggestion
class ArticleLikeCommentSerializer(serializers.ModelSerializer):
```

Reply:
게시글 목록 조회용 serializer와 게시글 상세 조회용 serializer를 구분하면 더 좋아. 각각의 경우에 필요한 데이터가 다르니까!